operator/v1/types_csi_cluster_driver.go

```diff
@@ -12,15 +12,14 @@ import (
 //     name: "ebs.csi.aws.com"
 //   spec:
 //     logLevel: Debug
-//     driverConfig:
-//       driverName: "ebs.csi.aws.com"
 
 // +genclient
 // +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // ClusterCSIDriver object allows management and configuration of a CSI driver operator
-// installed by default in OpenShift.
+// installed by default in OpenShift. Name of the object must be name of the CSI driver
+// it operates. See CSIDriverName type for list of allowed values.
```

Comment:
> See CSIDriverName type for list of allowed values.
Down below we remove the list of allowed values.

Reply:
if you mean `kubebuilder` comment, then that one would not be used anywhere, as there is no field with CSIDriverName type.

Reply:
Meant to say that I didn't find any "list of allowed values" cited in the comment.

Reply:
oh, nevermind; I found it